app/src/components/MemberDetail.vue

```diff
@@ -0,0 +1,14 @@
+<template>
+  <div className="card bg-white shadow-xl flex flex-row justify-between h-[83px] w-full">
+    <div className="card-body flex flex-col justify-center">
+      <p class="text-[#212B36]">{{ member.name }}</p>
+      <p class="text-[#212B36]">{{ member.address }}</p>
```

Comment:
This way of using the color is not good. 
It's don't allow your to handle colors globaly. 

Look at this her https://daisyui.com/docs/colors/